checkov/main.py

```diff
@@ -250,7 +251,10 @@ def run(self, banner: str = checkov_banner, tool: str = checkov_tool, source_typ
             "Python_implementation": platform.python_implementation()
         }
 
-        logger.debug(f'Run metadata: {json.dumps(self.run_metadata, indent=2)}')
+        logger.debug(f'Run metadata: {json.dumps(self.run_metadata, indent=2)}')  # noqa: E501
```

Comment:
why was this noqa added?

Reply:
originally added because flake8 was flagging this line as too long in my IDE. I have removed the noqa and edited my IDE settings instead for consistency